modules/core/src/layers/xviz-layer.js

```diff
@@ -378,6 +432,7 @@ export default class XVIZLayer extends CompositeLayer {
             data,
             lightSettings,
             wireframe: layerProps.stroked,
+            extruded: layerProps.stroked,
```

Comment:
This doesn't make sense to me. Do we know where this comes from?

Reply:
this could better check if we have a height instead, I will make the change